IpmiFeaturePkg/Test/Mock/Library/GoogleTest/MockIpmiCommandLib/MockIpmiCommandLib.inf

```diff
@@ -0,0 +1,36 @@
+## @file MockIpmiCommandLib.inf
+# Google Test mocks for IpmiCommandLib
+#
+# Copyright (c) Microsoft Corporation.
+# Your use of this software is governed by the terms of the Microsoft agreement under which you obtained the software.

```

Comment:
And here

  Copyright (c) Microsoft Corporation.
  SPDX-License-Identifier: BSD-2-Clause-Patent